core/src/empty/empty.module.css

```diff
@@ -0,0 +1,17 @@
+.container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    font-family: Inter;
+}
+
+.errorText {
+    font-style: normal;
+    font-weight: normal;
+    font-size: 14px;
+    line-height: 20px;
+    /* identical to box height, or 143% */
+    /* ent_color_neutral/navy500 */
+    color: #5E6D84;
```

Comment:
This color should belong to internal kit @thien-do

Reply:
Yeah, should use one of our css variables

Reply:
And btw I dont think we should use any color for the text here.. just the default black should be ok